package.json

```diff
@@ -14,6 +14,7 @@
     "migrate": "knex migrate:latest --knexfile src/knexfile.js",
     "rollback": "knex migrate:rollback --knexfile src/knexfile.js",
     "test": "NODE_ENV=test mocha --recursive --require babel-register test",
+    "prettier": "prettier --single-quote --trailing-comma es5 --write \"src/**/*.js\" \"test/**/*.js\"",
```

Comment:
I would not want a trailing comma, it looks alien to me even though it has it's benefits. And can you also increase the print width to 120 characters and re-run prettier again?

Reply:
Rename the key to `prettify` as well.

Reply:
We need to make an update here too: https://github.com/mesaugat/express-api-es6-starter/blob/master/package.json#L20-L25

Reply:
👍 on 120 lines and no trailing comma (only just got comfortable to both and prefer them, but I agree it looks awkward to most).

I don't think we need to change the `lint-staged` config. I have already added the `prettier` plugin to eslint. `prettier` is run as an eslint rule and can be applied as `eslint --fix`.

Reply:
Great 👍